README.md

```diff
@@ -206,6 +206,9 @@ If you are planning on building functions using the `dockerfile` template you ne
 #### Enable scaling to zero
 If you want your functions to scale to zero then you need to set `scale_to_zero: true`.
 
+#### Disable network policies
```

Comment:
`Toggle network policies`

Add comment:

```
# When deployed, network policies restrict communication so that functions cannot talk to the core OpenFaaS components in the `openfaas` namespace. They also prevent functions from invoking each other directly. It is recommended to enable this feature.
```

Reply:
Done 👍

Reply:
I can't see this comment?

Reply:
Done for real